charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```diff
@@ -114,3 +140,15 @@ spec:
       - name: kernel-modules
         hostPath:
           path: /lib/modules
+      - name: kube-proxy-cleanup-script
+        configMap:
+          name: kube-proxy-cleanup-script
+          defaultMode: 0777
+      - name: kube-proxy-dir
+        hostPath:
+          path: /var/lib/kube-proxy
+          type: DirectoryOrCreate
+      - name: kube-proxy-mode
+        hostPath:
+          path: /var/lib/kube-proxy/mode
+          type: FileOrCreate
```

Comment:
`kube-proxy-dir` is enough.
```suggestion
```

Reply:
It will not work if I skip creating the `/var/lib/kube-proxy/mode` file because `cat` will return an error for a non-existing file. This is the case if we create a new cluster.

Reply:
You can use 
```bash
touch "$1"
```
at the beginning of the cleanup script.

Reply:
Thats true, but I guess `FileOrCreate` function was exactly created for this purpose.

Reply:
I found it very strange to mount 2 different volumes just because of a single file, but I am fine.